src/api/appUsers.js

```diff
@@ -24,6 +24,30 @@ export class AppUsersApi extends BaseApi {
     });
   }
 
+  create(userId, props = {}) {
+    if (!userId || !userId.trim()) {
+      throw new Error('Must provide an user id.');
```

Comment:
You should name the parameter specifically in this error, eg `userId` not `user id`.